pkg/entities/feature.go

```diff
@@ -24,6 +24,8 @@ type Feature struct {
 	FeatureExperiments []Experiment
 	Rollout            Rollout
 	VariableMap        map[string]Variable
+	ExperimentRules    []Experiment
```

Comment:
ExperimentRules and DeliveryRules should be arrays of OptimizelyExperiment objects.

Reply:
Good catch! I need to delete these, as I am using OptimizelyFeature.